### PR TITLE
feature(EmotionFX): remove deprecated opengl render window

### PR DESCRIPTION
**Pull request by pollend**
This just removes the Opengl window from the EmotionFX module. Is there any functionality that should be carried over first before removing this. I left in a lot of the core code but this should be a gradual process with removing this so the individual PR are more easy to review.

ref: https://github.com/o3de/o3de/pull/9706

Signed-off-by: Michael Pollind <mpollind@gmail.com>

### CODE REVIEW

**Review by moraaar — APPROVED** [review]
LGTM

**Comment by hultonha** [comment]
Adding @amzn-rhhong and @amzn-jillich for visibility. I know this was something planned to be done soon and there's likely more bits and pieces to go to, want to make sure they're both aware. Thanks for looking at this though, super helpful!

**Review by amzn-jillich — APPROVED** [review]

**Review by cgalvan — APPROVED** [review]
Woo love to see red code :)

File: Gems/EMotionFX/Code/EMotionFX/Tools/EMotionStudio/EMStudioSDK/Source/PluginManager.cpp [hunk 1]
```diff
@@ -234,7 +233,6 @@ namespace EMStudio
         RegisterPlugin(new MotionSetsWindowPlugin());
         RegisterPlugin(new NodeGroupsPlugin());
         RegisterPlugin(new AnimGraphPlugin());
-        RegisterPlugin(new OpenGLRenderPlugin());
```
Comment (amzn-rhhong) [inline Gems/EMotionFX/Code/EMotionFX/Tools/EMotionStudio/EMStudioSDK/Source/PluginManager.cpp:237]: One thing worth to hand checking is what happen when loading a old layout still with OpenGLRenderPlugin. It should ignore the plugin with some kind of warning msg saying something like "openGLPlugin is disabled, switch to atomRenderPlugin instead.

For this PR i think the current stage is ok - as long as loading the old layout won't introduce a crash.

File: Gems/EMotionFX/Code/EMotionFX/Tools/EMotionStudio/EMStudioSDK/Source/Workspace.cpp [hunk 2]
```diff
@@ -393,18 +392,6 @@ namespace EMStudio
                 continue;
             }
 
-            // Temp solution after we refactor / remove the actor manager.
```
Comment (amzn-rhhong) [inline Gems/EMotionFX/Code/EMotionFX/Tools/EMotionStudio/EMStudioSDK/Source/Workspace.cpp:396]: GREAT! nice to see temp solution being removed.

**Comment by amzn-rhhong** [comment]
LGTM! @pollend Great job there! Echo on @amzn-jillich 's suggestion as tons of more code could be removed too. Good first step towards that goal :)

**Review by amzn-rhhong — APPROVED** [review]

**Comment by pollend** [comment]
what is this test for: MotionPlaybacksTests?

**Pull request merged**